packages/atomic/cypress/utils/mock-result.ts

```diff
@@ -0,0 +1,58 @@
+import {Raw, Result} from '@coveo/headless';
+
+export function buildMockResult(config: Partial<Result> = {}): Result {
```

Comment:
Copy and pasted from headless. Without mocking results, some tests failed frequently when testing locally.